public/v1/apps/elasticsearch.json

```diff
@@ -0,0 +1,83 @@
+{
+    "captainVersion": 1,
+    "documentation": "Elasticsearch (Single and Cluster). Taken from docker.elastic.co/elasticsearch/elasticsearch",
```

Comment:
This is a documentation field. I think your URL is just a link to the Docker image. Can you please replace it with the documentation for Elastic Search? This is for future maintenance... Specifically, we want to document where and how the following items are set:
- Environmental variables such as `discovery.zen.ping.unicast.hosts` , `ES_JAVA_OPTS`,  `cluster.name` and etc
- Volumes path such as `/usr/share/elasticsearch/data`